app/helpers/planning_application_helper.rb

```diff
@@ -19,6 +19,10 @@ def proposed_or_existing(planning_application)
     planning_application.work_status == "proposed" ? "No" : "Yes"
   end
 
+  def map_link(postcode)
+    "https://www.google.co.uk/maps/place/#{postcode.gsub(/\s+/, '')}"
```

Comment:
Sorry, should have realised this in my first review, but I think it would be best if we used `CGI::escape` here, so we a) maintain the space in case that's helpful, and b) automatically handle anything else weird that might happen here.

In fact, and just an idea that might not work, what happens if we pass in `CGI::escape(@planning_application.site.full_address)` - does that create a valid link to the exact address?

Reply:
I've tried this with a variety of weird and wonderful inputs and it seems to more or less find the exact address each time